Changelog.md

```diff
@@ -1,3 +1,8 @@
+Unreleased
+-------------------------
+* [Bug fix] make sure that the `stack_name` doesn't include any
```

Comment:
For the changelog, I'd just put "Restrict stack names to ASCII characters and digits." The detail about Nova and keypairs is already in the commit message, where it belongs.

Reply:
Ack, done.